platform_umbrella/apps/verify/test/traditional_test.exs

```diff
@@ -4,45 +4,20 @@ defmodule Verify.TraditionalTest do
 
   require Logger
 
-  @new_traditional_path "/traditional_services/new"
   @show_traditional_path ~r(/traditional_services/[\d\w-]+/show$)
 
-  @container_panel Query.css("#containers_panel-containers")
-  @port_panel Query.css("#ports_panel")
-
   verify "can create traditional service", %{session: session} do
     service_name = "int-test-#{:rand.uniform(10_000)}"
 
     session
-    # create service
-    |> visit(@new_traditional_path)
-    |> assert_has(h3("New Traditional Service"))
-    |> fill_in_name("service[name]", service_name)
-    # add container
-    |> find(@container_panel, fn e -> click(e, Query.button("Add Container")) end)
-    |> fill_in(Query.text_field("container[name]"), with: "echo")
-    |> fill_in(Query.text_field("container[image]"), with: @echo_server)
-    |> click(Query.css(~s/#container-form-modal-modal-container button[type="submit"]/))
-    # make sure the modal is gone
-    |> refute_has(Query.css("#container-form-modal"))
```

Comment:
In order to check that that the query really isn't found, it waits for `max_wait_time` for each `refute_has` :grimacing:

Reply:
TIL. Wow yeah that's not worth it.